app/components/stepper.js

```diff
@@ -9,9 +9,13 @@ import { JOIN_URL } from '../constants/apis';
 export default class StepperComponent extends Component {
   @service login;
   @service toast;
+  @service router;
   @tracked preValid = false;
   @tracked isValid = JSON.parse(localStorage.getItem('isValid')) ?? false;
-  @tracked currentStep = +localStorage.getItem('currentStep') ?? 0;
+  @tracked currentStep =
+    +localStorage.getItem('currentStep') ??
+    +new URLSearchParams(window.location.search).get('step') ??
```

Comment:
Use router service here to get query param.

Reply:
now I am using `this.args.step` for taking the value from param.